lib/bundle-validator.js

```diff
@@ -106,6 +107,119 @@ function validateThemeConfiguration(callback) {
     callback(null, true);
 }
 
+/**
+ * Find translatable strings
+ * @param {object} schema
+ * @param {function} callback
+ */
+function getTranslatableStrings(schema, callback) {
```

Comment:
What are your thoughts to keep some of these helper methods inside a subdirectory, this file can be about the process of running validations, less about the implementation details.

Reply:
Ok. I will do